### PR TITLE
Adding InMemoryLogger to the Getting Started Notebook

### PR DESCRIPTION
**Pull request by ishanashastri**
Stores training data in InMemoryLogger instances and plots the results to compare speed updates

### CODE REVIEW

**Review by ravi-mosaicml — APPROVED** [review]
LGTM, nice job! Though I am a bit concerned that the composer accuracy is _lower_ than baseline in the plot that this notebook generates. That's a separate issue which shouldn't hold up this PR, but we should investigate why or figure out how to fix it (do we need more epochs?)

**Comment by ishanashastri** [comment]
I think the reason for only doing 3 epochs on the baseline run is to demonstrate that 3 epochs running without Composer takes approximately the same amount of time as 4 epochs running with Composer. I can still modify the baseline if demonstrating that speedup effect is not as important, though.
